### PR TITLE
Fix Startup for Eco

### PR DESCRIPTION
**Pull request by gOOvER**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### CODE REVIEW

**Comment by gOOvER** [comment]
I hate GitKraken :/

**Comment by gOOvER** [comment]
Extra Flag added -> https://wiki.play.eco/en/Server_on_Linux

**Review by Software-Noob — APPROVED** [review]
LGTM, thank you for the contribution!

File: game_eggs/steamcmd_servers/eco/egg-eco.json [hunk 1]
```diff
@@ -77,6 +81,15 @@
             "user_viewable": true,
             "user_editable": false,
             "rules": "required|string|max:20"
+        },
+        {
+            "name": "Extra Flags",
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/eco/egg-eco.json:86]: This isn't actually applied anywhere in the egg?

**Review by Software-Noob — CHANGES_REQUESTED** [review]
Please resolve the extra flags as mentioned above

File: game_eggs/steamcmd_servers/eco/egg-eco.json (hunk 1 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/eco/egg-eco.json:86]: It is. It's Parkers Steam DL Script.

See https://github.com/parkervcp/eggs/blob/77317bb22cc3de258b17639fa271d4db168e0167/game_eggs/steamcmd_servers/eco/egg-eco.json#L21

**Comment by gOOvER** [comment]
> Please resolve the extra flags as mentioned above

Please take a look into Installscript. All there ;)

**Review by Software-Noob — APPROVED** [review]

**Pull request merged**